src/main/resources/org/openjdk/jextract/impl/resources/Messages.properties

```
@@ -83,6 +87,12 @@ Option                             Description
 \                                   is not specified, then unnamed package is used.             \n\
 --version                          print version information and exit                           \n
 
+jextract.usage.mac=\
+Platform dependent options for running jextract                                                 \n\
+--mac-framework-dir <dir>          specify the framework directory                              \n\
+-f <framework>                    specify framework library. -f libGL is equivalent to         \n\
+\                                      -l :/System/Library/Frameworks/libGL.framework/libGL         \n
```

Wrong / inconsistent indentation?
```suggestion
-f <framework>                     specify framework library. -f libGL is equivalent to         \n\
\                                   -l :/System/Library/Frameworks/libGL.framework/libGL       \n
```

---

Thanks. Fixed the first line in [9ce83f3](https://github.com/openjdk/jextract/pull/268/commits/9ce83f363e50b6e7f3fe5dae1612b929b38a7bd3), the spacing in the second line is intentional.